app/api/dao/user.py

```diff
@@ -9,12 +9,25 @@
 from app.utils.validation_utils import is_email_valid
 
 class UserDAO:
+    """Data Access Object for User functionalities"""
+    
     FAIL_USER_ALREADY_EXISTS = "FAIL_USER_ALREADY_EXISTS"
     SUCCESS_USER_CREATED = "SUCCESS_USER_CREATED"
     MIN_NUMBER_OF_ADMINS = 1
 
     @staticmethod
     def create_user(data):
+        """Creates a new user.
+        
+        Creates a new user with provided data.
+        
+        Arguments:
+            data: A list containing the user's name, username, password, and email, as well as recognition that they have read and agree to the Terms and Conditions.
+            
+        Returns:
+            A list with two elements. The first element is a dictionary containing a key 'message' containing a string which indicates whether or not the user was created successfully. The second is the HTTP response code.
```

Comment:
```suggestion
            A tuple with two elements. The first element is a dictionary containing a key 'message' containing a string which indicates whether or not the user was created successfully. The second is the HTTP response code.
```

What is returned is a tuple and not a list.